examples/USGS/USGS_NLCD_RELEASES_2023_REL_RCMAP_V6_TRENDS_preview.js

```diff
@@ -0,0 +1,54 @@
+// Import the RCMAP TRENDS image.
+var image = ee.Image('USGS/NLCD_RELEASES/2023_REL/RCMAP/V6/TRENDS')

```

Comment:
ditto.

Reply:
Same thing here. When I comment on Line 2 and add this in for Line 3 (pointing to the Local Source - which is set to Anyone can read) it works fine for me. 
var image = ee.Image('projects/usgs-gee-audra-griebel/assets/eros/rcmap_v6_trends/TRENDS')
It's supposed to be an Image for the Trends non-year.